litmus-portal/frontend/src/components/Header/ProfileDropdownSection.tsx

```diff
@@ -30,17 +31,7 @@ const ProfileDropdownSection: React.FC<ProfileInfoDropdownSectionProps> = ({
   const [isProfilePopoverOpen, setProfilePopoverOpen] = useState(false);
 
   const profileMenuRef = useRef();
-
-  let initials = ' ';
-
-  if (name) {
-    const nameArray = name.split(' ');
-
-    initials =
-      nameArray[0][0].toUpperCase() +
-      nameArray[nameArray.length - 1][0].toUpperCase();
-  }
-
+  const initials = userAvatar(name, false);
   const sendSelectedProjectIDToHeader = (selectedProjectID: any) => {
```

Comment:
You don't need to specify `selectedProjectID: any` since you have already declared it as a `string` in the interface. Use = (selectedProjecID) => instead

Reply:
(`selectedProjectID: string)`  had to put string, otherwise it was showing error.